KomaMRIBase/src/datatypes/Phantom.jl

```diff
@@ -612,5 +535,61 @@ function check_phantom_arguments(nd, ss, us)
             ssy = ss[2]
         end
     end
+    
     return ssx, ssy, ssz, usx, usy, usz
 end
+
+"""
+    T1, T2, T2s, ρ, Δw = default_brain_tissue_properties(labels, tissue_properties = nothing)
+
+This function returns the default brain tissue properties using a labels identifier Matrix
+# Arguments
+- `labels` : (`::Matrix`) the labels identifier matrix of the phantom
+- `tissue_properties` : (`::Dict`, `=Dict()`) phantom tissue properties in ms and Hz considering the available tissues
+
+# Returns
+- `T1, T2, T2s, ρ, Δw`: (`::Matrix`) matrices of the same size of labels with the tissues properties information
+
+# Examples
+```julia-repl
+julia> T1, T2, T2s, ρ, Δw = default_brain_tissue_properties(labels, tissue_properties)
+
+julia> T1, T2, T2s, ρ, Δw = default_brain_tissue_properties(labels)
+```
+"""
+function default_brain_tissue_properties(labels, tissue_properties = Dict())
+    # Load default tissue properties
+    default_properties = Dict(
+        # T1, T2, T2*, ρ, Δw
+        "CSF" => [2.569, 0.329, 0.058, 1, 0],
+        "GM" => [0.833, 0.083, 0.069, 0.86, 0],
+        "WM" => [0.500, 0.070, 0.061, 0.77, 0],
+        "FAT1" => [0.350, 0.070, 0.058, 1, -3.84], #-220 Hz
+        "MUSCLE" => [0.900, 0.047, 0.030, 1, 0],
+        "SKIN/MUSCLE" => [0.569, 0.329, 0.058, 1, 0],
+        "SKULL" => [0, 0, 0, 0, 0],
+        "VESSELS" => [0, 0, 0, 0, 0],
+        "FAT2" => [0.500, 0.070, 0.061, 0.77, -3.84], #-220 Hz
+        "DURA" => [2.569, 0.329, 0.058, 1, 0],
+        "MARROW" => [0.500, 0.070, 0.061, 0.77, 0])
```

Comment:
Improve indentation, and reorder properties (not done here):
```suggestion
    default_properties = Dict(
        #            T1      T2     T2*     ρ     Δw
        "CSF"         => [2.569, 0.329, 0.058, 1,    0],
        "GM"          => [0.833, 0.083, 0.069, 0.86, 0],
        "WM"          => [0.500, 0.070, 0.061, 0.77, 0],
        "FAT1"        => [0.350, 0.070, 0.058, 1,   -3.84], # -220 Hz
        "MUSCLE"      => [0.900, 0.047, 0.030, 1,    0],
        "SKIN/MUSCLE" => [0.569, 0.329, 0.058, 1,    0],
        "SKULL"       => [0,     0,     0,     0,    0],
        "VESSELS"     => [0,     0,     0,     0,    0],
        "FAT2"        => [0.500, 0.070, 0.061, 0.77, -3.84], # -220 Hz
        "DURA"        => [2.569, 0.329, 0.058, 1,    0],
        "MARROW"      => [0.500, 0.070, 0.061, 0.77, 0]
    )
```